src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,93 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+    
+    private final static ArrayList<Tagging> ALL_TAGGING_CHANGES = new ArrayList<Tagging>();
```

Comment:
this implementation would be case that was specified in the second diagram of the exercise question. Where else can be a better place to store all the tagging changes and why? 

Hint: look at the first diagram of the exercise question.